CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+
+## I Want To Contribute
+
+### Reporting Bugs
+
+#### Before Submitting a Bug Report
+
+A good bug report shouldn't leave others needing to chase you up for more information. Therefore, we ask you to investigate carefully, collect information and describe the issue in detail in your report. Please complete the following steps in advance to help us fix any potential bug as fast as possible.
+
+- Make sure that you are using the latest version.
+- Determine if your bug is really a bug and not an error on your side e.g. using incompatible environment components/versions (Make sure that you have read the [README](https://github.com/nimblehq/flutter-templates/blob/main/README.md). If you are looking for support, you might want to check [this section](#i-have-a-question)).
+- To see if other users have experienced (and potentially already solved) the same issue you are having, check if there is not already a bug report existing for your bug or error in the [bug tracker](https://github.com/nimblehq/flutter-templates/issues?q=is%3Aissue+label%3A%22type+%3A+bug%22+).
+- Also make sure to search the internet (including Stack Overflow) to see if users outside of the GitHub community have discussed the issue.
+- Collect information about the bug:
+  - Stack trace (Traceback)
+  - OS, Platform and Version (Windows, Linux, macOS, x86, ARM)
+  - Flutter's doctor (if the generated project runs failed)
+  - Possibly your input and the output
+  - Can you reliably reproduce the issue? And can you also reproduce it with older versions?
+
+#### How Do I Submit a Good Bug Report?
+
+> You must never report security related issues, vulnerabilities or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead sensitive bugs must be sent by email to <dev@nimblehq.co>.
+
+We use GitHub issues to track bugs and errors. If you run into an issue with the project:
+
+- Open a [Bug Report's Issue](https://github.com/nimblehq/flutter-templates/issues/new?assignees=&labels=type+%3A+bug&projects=&template=bug_template.md&title=Fix%3A+).
+- Explain the behavior you would expect and the actual behavior.
+- Please provide as much context as possible and describe the *reproduction steps* that someone else can follow to recreate the issue on their own. This usually includes your code. For good bug reports you should isolate the problem and create a reduced test case.
+- Provide the information you collected in the previous section.
+
+Once it's filed:
+
+- The repository's maintainer will mark the issue as `triage` and try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for further information. Bugs with the `triage` label will not be addressed until they are reproduced.
+- If the team is able to reproduce the issue, the `triage` label will be removed, and the issue will be left to be [implemented by someone](#your-first-code-contribution).
+
```

Comment:
Same, we should not provide a commitment and our process here. We might not have enough time to maintain them all the time due to client projects 🙏 

```suggestion
```